src/components/views/ViewDataTable/ViewDataTableToolbar.tsx

```diff
@@ -1,31 +1,65 @@
-import { FormattedMessage } from 'react-intl';
-import { Add, Launch } from '@material-ui/icons';
-import { Box, Button } from '@material-ui/core';
+import { useContext } from 'react';
+import { Add, Launch, RemoveCircleOutline } from '@material-ui/icons';
+import { Box, Button, Slide, Tooltip } from '@material-ui/core';
+import { FormattedMessage, useIntl } from 'react-intl';
 
+import { ConfirmDialogContext } from 'hooks/ConfirmDialogProvider';
 
 export interface ViewDataTableToolbarProps {
+    disabled: boolean;
+    isSmartSearch: boolean;
     onColumnCreate: () => void;
+    onRowsRemove: () => void;
     onViewCreate: () => void;
     selection: number[];
 }
 
 const ViewDataTableToolbar: React.FunctionComponent<ViewDataTableToolbarProps> = ({
+    disabled,
+    isSmartSearch,
     onColumnCreate,
+    onRowsRemove,
     onViewCreate,
     selection,
 }) => {
+    const intl = useIntl();
+    const { showConfirmDialog } = useContext(ConfirmDialogContext);
+
+    const onClickRemoveRows = () =>{
+        showConfirmDialog({
+            onSubmit: onRowsRemove,
+            title: intl.formatMessage({ id: 'misc.views.removeDialog.title' }),
+            warningText: intl.formatMessage({ id: 'misc.views.removeDialog.action' }),
+        });
+    };
+
     return (
         <Box display="flex" justifyContent="flex-end">
-            { !!selection.length && (
+            <Slide direction="left" in={ !!selection.length } timeout={ 150 }>
```

Comment:
I tried a few different ways but I liked this the best - maybe we can just leave it as-is for now? Since we might redesigning the toolbar anyway - based on space considerations and perhaps Gemma fleshing out designs at some point?